pyproject.toml

```diff
@@ -69,6 +69,36 @@ mkdocs-material = "^5.5.5"
 # HACK https://readthedocs.org/api/v2/build/11653917.txt
 # TODO Remove this dependency when mkdocstrings supports pymdown-extensions==8.0
 pymdown-extensions = "<8.0,>=6.3"
+tox = "^3.19.0"
+
+[tool.poe.tasks]
+clean.sequence = ["_clean-build", "_clean-pyc"]
+clean.help = "remove build/python artifacts"
+_clean-build = """
+rm -rf build/
+       dist/
+       *.egg-info
+       pip-wheel-metadata
+       *.egg-info
+"""
+_clean-pyc = """
+rm -rf .pytest_cache
+       ./**/__pycache__
+       ./**/*.pyc
+       ./**/*.pyo
+       ./**/*~
+"""
+
+docs = "mkdocs serve"
+
+test.cmd = "pytest -x"
+test.help = "run tests"
+
+check.cmd = "pre-commit run --all-files"
+check.help = "check (and auto-fix) style with pre-commit"
```

Comment:
```suggestion
lint.cmd = "pre-commit run --all-files"
lint.help = "check (and auto-fix) style with pre-commit"
```

Reply:
that command does a lot more than linting?

Reply:
Well, it lints and reformats IIRC.